calour/io.py

```diff
@@ -211,9 +212,12 @@ def read_taxa(data_file, sample_metadata_file=None,
 
     Returns
     -------
-    exp : Experiment
+    exp : ``AmpliconExperiment``
+        after removing low read sampls and normalizing
     '''
-    exp = read(data_file, sample_metadata_file, **kwargs)
+    data, sample_metadata, feature_metadata, exp_metadata, description = _read(data_file, sample_metadata_file, **kwargs)
+    exp = AmpliconExperiment(data, sample_metadata, feature_metadata,
+                             exp_metadata=exp_metadata, description=description, sparse=sparse)
 
     exp.feature_metadata.index = exp.feature_metadata.index.str.upper()
```

Comment:
to prevent uppercase/lowercase problems in comparison. we convert all sequences to upper